app/src/shared/components/SvgIcon/index.jsx

```diff
@@ -174,8 +174,8 @@ const sources = {
                     <path
                         d="M517,2614.5 C516.792893,2614.5 516.625,2614.66789 516.625,2614.875 C516.625,2615.08211 516.792893,2615.25 517,2615.25 C517.207107,2615.25 517.375,2615.08211 517.375,2614.875 C517.375,2614.66789 517.207107,2614.5 517,2614.5"
                         id="Path"
-                        strokeLineCap="round"
-                        strokeLineJoin="round"
+                        strokeLinecap="round"
+                        strokeLinejoin="round"
```

Comment:
🤔

Reply:
See my note above about my bad habit of adding unrelated PR code:
* includes strokeLinejoin, strokeLinecap typos from previous docs PR

Reply:
Ah, missed that! Good spotting on those